docs/general/ad-filtering/create-own-filters.md

```diff
@@ -353,8 +353,8 @@ The modifier `":" h_value` part may be omitted. In that case the modifier matche
 * [`$ping`](#ping-modifier)
 * [`$xmlhttprequest`](#xmlhttprequest-modifier)
 * [`$websocket`](#websocket-modifier)
-* [`$webrtc`](#webrtc-modifier)
 * [`$other`](#other-modifier)
+* [`$webrtc` (deprecated)](#webrtc-modifier)
```

Comment:
let's place it after `$object-subrequest` to keep the alphabetical order

Reply:
and place `$other` on the top of the list

Reply:
@el-termikael 
Do you mean this order:
- other
- object-subrequest
- webrtc
?

Reply:
@el-termikael this whole list is not in alphabetical order, for example, we have modifiers like $object at the very top.
We don't need to move $other above, because this modifier is not widely used.

Reply:
current order may remain as is